pages/21.Video/docs.en.md

```diff
@@ -46,6 +46,7 @@ URL to the actual mp4 file location in order to use the gated video feature.
     <source type="video/youtube" src="https://www.youtube.com/watch?v=QT6169rdMdk" />
 </video>
 ```
+> Note: Mautic only supports `<video>` elements with proper `<source>` elements in it for the Gated video functionality. Mautic ignores all video tags that don't have the `data-form-id` and `data-gate-time` attributes
```

Comment:
I think that this may not need to be capitalised in the middle of the sentence - what do you think?

```suggestion
> Note: Mautic only supports `<video>` elements with proper `<source>` elements in it for the gated video functionality. Mautic ignores all video tags that don't have the `data-form-id` and `data-gate-time` attributes
```

Reply:
I actually capitalized it to make it a useful sign for the reader, but I don't think i should as it's already under Note